idea/testData/intentions/branched/safeAccessToIfThen/letTopLevelVar.kt.after

```diff
@@ -0,0 +1,7 @@
+// WITH_RUNTIME
+
+var a: String? = "A"
+fun main(args: Array<String>) {
+    val a1 = a
+    if (a1 != null) a1.length + 1
+}
```

Comment:
I'd add at least one test with regular (non-redundant) `let` and more tests with redundant `let`, e.g. `a?.let { foo(it) }` and may be something else.